src/components/Timeline/updates/TimelineJourneyMilestone.tsx

```diff
@@ -0,0 +1,127 @@
+import CancelIcon from '@material-ui/icons/Cancel';
+import CheckCircleIcon from '@material-ui/icons/CheckCircle';
+import FlagIcon from '@material-ui/icons/Flag';
+import { FormattedMessage } from 'react-intl';
+import React from 'react';
+import { Box, Grid, Typography } from '@material-ui/core';
+
+import PersonHoverCard from 'components/PersonHoverCard';
+import theme from 'theme';
+import TimelineActor from '../TimelineActor';
+import { ZetkinPerson as ZetkinPersonType } from 'types/zetkin';
+import ZetkinRelativeTime from 'components/ZetkinRelativeTime';
+import {
+  CHANGE_PROPS,
+  UPDATE_TYPES,
+  ZetkinUpdateJourneyMilestone,
+} from 'types/updates';
+
+const getPersonName = (person: Partial<ZetkinPersonType>) =>
+  person.first_name + ' ' + person.last_name;
```

Comment:
This would be nice to put in `utils` somewhere, maybe `stringUtils`?

Reply:
That's a job for another issue which I believe exists already somewhere. The need for a global display name function that works according to the locale as well... 
This is a stopgap until that issue is picked up by someone.